editors/linux-wps-office/Makefile

```diff
@@ -0,0 +1,55 @@
+PORTNAME=	wps-office
+DISTVERSION=	11.1.0.10161
+CATEGORIES=	editors linux
+MASTER_SITES=	https://wdl1.pcfg.cache.wpscdn.com/wpsdl/wpsoffice/download/linux/10161/
+PKGNAMEPREFIX=	linux-
+DISTNAME=	${PORTNAME}-${PORTVERSION}.XA-1.x86_64
+
+MAINTAINER=	ygy@FreeBSD.org
+COMMENT=	Kingsoft WPS Office free for Linux
+
+LICENSE=	commercial
+LICENSE_NAME=	End User License Agreement
+LICENSE_TEXT=	${RESTRICTED}
+LICENSE_PERMS=	no-auto-accept no-dist-mirror no-dist-sell no-pkg-mirror no-pkg-sell
+
+RESTRICTED=	Source recipient must acknowledge the license. \
+		Reproduction or redistribution prohibited. \
+		See https://www.wps.com/eula.
+
+USES=		desktop-file-utils linux shebangfix
+INSTALLS_ICONS=	yes
+USE_LINUX=	png pulseaudio-libs xorglibs
+
+SHEBANG_FILES= 	usr/bin/et \
+		usr/bin/wps \
+		usr/bin/wpspdf \
+		usr/bin/wpp
+
+ONLY_FOR_ARCHS= amd64
+
+EXTRACT_SUFX=   .rpm
+NO_BUILD=	yes
+NO_WRKSUBDIR=	yes
+PLIST_SUB=	LINUXBASE=${LINUXBASE}
```

Comment:
Maybe we can move this to Uses/linux.mk and clean up other ports :)

Reply:
Yeah, we could potentially do this in another PR/Phabricator review. I am trying hard to think of a counterexample why we shouldn't do this, but can't.

Reply:
Yes no need to do this immediately, but is a good thing to do after adding this port. :)